proxy/Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM registry.access.redhat.com/ubi9/ubi:9.4 as builder
```

Comment:
Can I ask you to do the same for `.devcontainer/Dockerfile`? It is the same `Node.js` binary. Just to make commit looks better.

Reply:
@ai I made a change, but could you run it locally please, because I could not verify `.devcontainer/Dockerfile` on my machine (I suppose because of the arm arch)